<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelegramBot</class>
 <widget class="QMainWindow" name="TelegramBot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>388</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>724</width>
    <height>388</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>724</width>
    <height>388</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>370</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>370</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="api_id_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>5</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>API ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="chats_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>155</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Чаты</string>
       </property>
      </widget>
      <widget class="QLabel" name="api_hash_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>55</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>API Hash</string>
       </property>
      </widget>
      <widget class="QLabel" name="keywords_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>204</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ключевые слова</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="api_id_ledit">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>20</y>
         <width>250</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="chats_ledit">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>170</y>
         <width>250</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="keywords_ledit">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>220</y>
         <width>250</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="api_hash_ledit">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>70</y>
         <width>250</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="destination_ledit">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>120</y>
         <width>250</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="destination_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>104</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Получатель</string>
       </property>
      </widget>
      <widget class="QPushButton" name="login_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>330</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Авторизация</string>
       </property>
      </widget>
      <widget class="QLabel" name="tel_num_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>255</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Номер телефона</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="tel_num_ledit">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>270</y>
         <width>250</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>370</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>370</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QListWidget" name="info_list">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>350</width>
         <height>295</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="start_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>330</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Старт парсинга</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
